src/View3d.ts

```diff
@@ -135,10 +136,7 @@ export class View3d {
    * @param {VolumeChannelDisplayOptions} options
    */
   setVolumeChannelOptions(volume: Volume, channelIndex: number, options: VolumeChannelDisplayOptions): void {
-    if (!this.image) {
-      return;
-    }
-    this.image.setChannelOptions(channelIndex, options);
+    this.image?.setChannelOptions(channelIndex, options);
```

Comment:
these are incurring a functionality change: when this.image is null, redraw will be called now where it wasn't before.